.github/workflows/prod-build.yml

```
@@ -310,36 +317,53 @@ jobs:
           # Info about which CONTENT_* environment variables were set and to what.
           echo "CONTENT_ROOT=$CONTENT_ROOT"
           echo "CONTENT_TRANSLATED_ROOT=$CONTENT_TRANSLATED_ROOT"
-          yarn build:sw
-          yarn build:client
-          yarn build:ssr
-
-          cp assets/prod/robots.txt client/build/robots.txt
 
-          yarn rari content sync-translated-content
-          yarn rari git-history
+          mkdir -p out/static
```

Nit: Use `$BUILD_OUT_ROOT`.

```suggestion
          mkdir -p "$BUILD_OUT_ROOT/static"
```